test/product.js

```diff
@@ -0,0 +1,74 @@
+let mongoose = require("mongoose");
+let Product = require('../dist/models/product');
+
+let chai = require('chai');
+let chaiHttp = require('chai-http');
+let server = require('../dist/server');
+let should = chai.should();
+
+
+chai.use(chaiHttp);
+
+describe('Product', () => {
+    beforeEach((done) => {
+        Product.default.remove({}, (err) => {
```

Comment:
it gave me a deprecation warning - deleteOne or deleteMany might be better option (not sure it's chai thing)